### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by fakeid30** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#317] [Ideas]
@cullzie should we cut a new release ? The default master branch is pretty outdated by now.

**Comment by cullzie** [comment]
@fakeid30 Ye I think we should. I'll try do it early this week. Think there is a alpha branch reference still on the README. I haven't got around to removing it yet 🙄

**Reply by fakeid30** [reply]
What about the pull requests?

**Reply by cullzie** [reply]
I'll leave a last comment on each and close at end of September if no progress.

The stats one is mostly implemented from my changes a while back. 
The freebiesglobal is totally new functionality. I can merge it and fix it if we don't see any progress from the contributor

**Comment by fakeid30** [comment]
Went ahead and removed alpha references from the readme in the develop branch.

**Comment by cullzie** [comment]
V3.2.0 Released on PyPi: https://pypi.org/project/udemy-enroller/3.2.0/
